docs/ECMAScriptModules.md

```diff
@@ -41,6 +45,12 @@ import.meta.jest.useFakeTimers();
 
 // jest === import.meta.jest => true
 ```
+:::note
```

Comment:
Not needed as well. The recently added Module mocking in ESM section (below) replaced this node.